roles/validations/README.md

```diff
@@ -0,0 +1,14 @@
+# validations
+The QE Validations role provides a framework for performing state verification.
+This is useful for QE purposes and will be leveraged post-deployment to assert
+that required conditions have been satisfied. For example, if I update a certain
+value in my `OpenStackDataPlaneNodeSet`. I want to validate that the change has
+been applied as per my expectations on the relevant OpenStack Compute nodes.
+
+## Privilege escalation
+If apply, please explain the privilege escalation done in this role.
+
+## Parameters
+* `param_1`: this is an example
+
+## Examples
```

Comment:
Do you want to show some usage examples? Might be good, once there's a working flow.

Reply:
Yeah, I completely forgot this file existed. Easter was too long apparently. :) 

I'll fix this file up

Reply:
thanks!